cmd/server-main.go

```diff
@@ -262,9 +274,8 @@ func serverMain(ctx *cli.Context) {
 
 	newObject, err := newObjectLayer(globalEndpoints)
 	if err != nil {
-		logger.LogIf(context.Background(), err)
-		err = globalHTTPServer.Shutdown()
-		logger.LogIf(context.Background(), err)
+		logger.FatalIf(err, "Unable to initialize data backend")
```

Comment:
This would have already killed the server... there is no reason to have the code after this line.

Reply:
s/Unable to initialize data backend/Unable to initialize backend/  is fine..

Reply:
> This would have already killed the server... there is no reason to have the code after this line.
Done

> s/Unable to initialize data backend/Unable to initialize backend/ is fine..
Done